src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+
+  if (days >= 3 && days < 7) {
+    return days * rentPrice - minDiscount;
+  } else if (days >= 7) {
+    return days * rentPrice - maxDiscount;
+  } else {
+    return days * rentPrice;
+  }
```

Comment:
you can make code simpler with two `ifs` and without `days < 7`, make check `days >= 3` after `days >= 7`

Reply:
fixed